packages/webapp/src/hooks/useScrollBox.ts

```diff
@@ -1,102 +1,91 @@
-import { useState, useEffect, useCallback } from "react";
-import throttle from "lodash/throttle";
+import { useState, useEffect, useCallback } from "react"
+import throttle from "lodash/throttle"
 
-const timing = (1 / 60) * 1000;
-const decay = (v: any) => -0.1 * ((1 / timing) ^ 4) + v;
+const timing = (1 / 60) * 1000
+const decay = (v: any) => -0.1 * ((1 / timing) ^ 4) + v
 
 function useScrollBox(scrollRef: any) {
-  const [clickStartX, setClickStartX] = useState();
-  const [scrollStartX, setScrollStartX] = useState();
-  const [isDragging, setIsDragging] = useState(false);
-  const [direction, setDirection] = useState(0);
-  const [momentum, setMomentum] = useState(0);
-  const [lastScrollX, setLastScrollX] = useState(0);
-  const [speed, setSpeed] = useState(0);
+  const [lastScrollX, setLastScrollX] = useState(0)
+  const [showLeftArrow, setShowLeftArrow] = useState<boolean>(false)
+  const [showRightArrow, setShowRightArrow] = useState<boolean>(false)
 
-  const scrollWrapperCurrent = scrollRef.current;
+  const scrollWrapperCurrent = scrollRef.current
+
+  const cardWidth = 200 // width of card when not in focus
+  const scrollAmount = 2 * cardWidth
+  const duration = 300
+
+  const checkArrowsVisibility = () => {
+    if (!scrollRef.current) return
+    const { scrollLeft, scrollWidth, clientWidth } = scrollRef.current
+    setShowLeftArrow(scrollLeft > 0)
+    setShowRightArrow(scrollLeft < scrollWidth - clientWidth)
+  }
```

Comment:
This needs to be called when the window resizes, otherwise if the arrows are not visible (e.g. window full screen) but you resize, then the arrows are not available anymore.

You can use `const playerHand = [0n, 1n, 2n, 3n, 4n, 5n, 6n, 7n, 8n, 9n]` to test (adapt the number of cards to your screen size!)

Reply:
Arrows now show up in the hand drawer when the window is resized.

Reply:
Support for scrolling with mouse wheel added, to be tested.